test/helpers/cilium.go

```diff
@@ -527,6 +529,11 @@ func (s *SSHMeta) PolicyWait(revisionNum int) *CmdRes {
 // ReportFailed gathers relevant Cilium runtime data and logs for debugging
 // purposes.
 func (s *SSHMeta) ReportFailed(commands ...string) {
+	if config.CiliumTestConfig.HoldEnvironment {
+		ginkgoext.GinkgoPrint("Skipped gathering logs (-cilium.holdEnvironment=true)\n")
```

Comment:
`ginkgoext.GinkgoPrint` didn't exist until commit c9492c4e15af ("Test: Improved Ginkgo logs"), which as far as I can tell was never in v1.0. This would need to be `fmt.Fprintf(wr, ...` instead.